src/hotspot/share/opto/c2_globals.hpp

```diff
@@ -633,6 +633,9 @@
   develop(bool, VerifyLoopOptimizations, false,                             \
           "verify major loop optimizations")                                \
                                                                             \
+  develop(bool, VerifyNoNewIrreducibleLoops, false,                         \
```

Comment:
Do you plan to add it to our stress testing?

Reply:
We cannot currently do that:

> With the new debug flag -XX:+VerifyNoNewIrreducibleLoops, we still hit the assert, as expected:

And we can do this later:

> For now, have the assert behind a Verify flag, so that [JDK-8348570](https://bugs.openjdk.org/browse/JDK-8348570) is unblocked. Later, we can remove the Verify flag and alway enable the assert again.

Does that sound ok to you?

Reply:
Sounds good. My question was about your future plan when "everything" is fixed.